code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -61,20 +61,30 @@
 /datum/gear/accessory/tie/New()
 	..()
 	var/ties = list()
-	ties["blue tie"] = /obj/item/clothing/accessory/blue
-	ties["red tie"] = /obj/item/clothing/accessory/red
 	ties["horrible tie"] = /obj/item/clothing/accessory/horrible
 	ties["blue tie with a clip"] = /obj/item/clothing/accessory/tie/blue_clip
 	ties["blue long tie"] = /obj/item/clothing/accessory/tie/blue_long
 	ties["red tie with a clip"] = /obj/item/clothing/accessory/tie/red_clip
 	ties["red long tie"] = /obj/item/clothing/accessory/tie/red_long
-	ties["black tie"] = /obj/item/clothing/accessory/tie/black
 	ties["dark green tie"] = /obj/item/clothing/accessory/tie/darkgreen
-	ties["yellow tie"] = /obj/item/clothing/accessory/tie/yellow
-	ties["navy tie"] = /obj/item/clothing/accessory/tie/navy
-	ties["white tie"] = /obj/item/clothing/accessory/tie/white
 	gear_tweaks += new/datum/gear_tweak/path(ties)
 
+/datum/gear/accessory/rectie
+	display_name = "colorful tie"
```

Comment:
If this is a recolorable tie, this name seems rather misleading. It's not a multicolor (i.e. colorful) tie, it's a recolorable tie.